src/cargo/ops/registry.rs

```diff
@@ -32,9 +33,13 @@ pub fn publish(manifest_path: &Path,
                config: &Config,
                token: Option<String>,
                index: Option<String>,
-               verify: bool) -> CargoResult<()> {
+               verify: bool, allow_untracked: bool) -> CargoResult<()> {
     let pkg = try!(Package::for_path(&manifest_path, config));
 
+    if !allow_untracked{
```

Comment:
Stylistically a space typically comes before the `{` here (also a few other places in this diff)